.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    env: {
+        browser: true,
+        es2021: true,
+    },
+    extends: [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended",
+    ],
+    overrides: [],
```

Comment:
**thought:** Looks good - we will probably have to adjust it a little bit this config later when we add code to it